style.css

```diff
@@ -261,22 +265,149 @@ body.lock {
     font-size: 20px;
     right: 25px;
 }
-.arrow_left {
+.arrow-left {
     border: solid var(--arrow-color);
     border-width: 0 2px 2px 0;
     display: inline-block;
     padding: 4px;
     transform: rotate(135deg);
 }
-.arrow_right {
+.arrow-right {
     border: solid var(--arrow-color);
     border-width: 0 2px 2px 0;
     display: inline-block;
     padding: 4px;
     transform: rotate(-45deg);
 }
 .content {
-    padding: 70px 0 0;
+    margin: 60px 0 0;
+}
+.carousel {
+    position: relative;
+    width: 100vw;
+    height: 150vw;
+    overflow: hidden;
+}
+.carousel__text {
+    position: absolute;
+    top: 50%;
+    left: 8%;
+    text-align: left;
+    display: active;
+    z-index: 20;
+}
+.carousel__text1 {
+    color: var(--element-color);
+    font-size: 9vw;
+    letter-spacing: 3px;
+}
+.carousel__text2 {
+    color: var(--additional-color);
+    font-size: 8vw;
+    letter-spacing: 1.5px;
+}
+.carousel__box {
+    position: absolute;
+    transition: 0.5s;
+    width: 100vw;
+}
+.carousel__object {
+    width: 100vw;
+    left: 0;
+    top: 0;
+}
+.carousel__buttons {
+    position: absolute;
+    top: 70%;
+    left: 8%;
+    background-color: transparent;
+    z-index: 10;
+}
+.carousel__button-left, .carousel__button-right {
+    height: 60px;
+    width: 60px; 
+    border: 3px solid var(--element-color);  
+    background-color: transparent;
+    margin-right: 10px;
+}
+.arrow_carousel {
+    padding: 6px;
+    border: solid var(--element-color);
+    border-width: 0 3px 3px 0;
+}
+.about-part {
+    position: static;
+    margin: 30px 10px; 
+    overflow: auto; 
+}
+.about-part__img-container {
+    position: relative;
+    height: 330px;
+}
+.img1 {
+    position: absolute;
+    left: 0;
+    top: 140px;
+    z-index: 10;
+    height: 150px;
+}
+.img2 {
+    position: absolute;
+    right: 0; 
+    top: 0; 
+    height: 150px;
+}
+.title {
+    font-size: 35px;
+    letter-spacing: 8px;
+}
+.subtitle {
+    color: var(--additional-color);
+    font-family: var(--main-text-font);
+    font-size: 12px;
+    letter-spacing: 1px;
+    text-transform: uppercase;
+}
+p {
```

Comment:
Why do you overwrite my paragraph styles? I needed my div to have default font size, and yet here you just overwrote my styles...

Reply:
it happened

Reply:
Rename this to a class

Reply:
Ok, added class

Reply:
I can still see that it's assigning style to paragraph not a class here.

Reply:
I pushed everything, it should be visible